ui/src/utils/fetchWrappers/piplelineVertexMetricsFetch.test.ts

```diff
@@ -0,0 +1,76 @@
+// Tests pipelineVertexMetricsFetch.ts
```

Comment:
we are using this API call inside `fetcherHooks/pipelineViewFetch.ts`. can we please move the tests there, and clean up this file

Reply:
I don't understand.  The test file is where the original file is. Do you want me to move the original file to fetcherHooks too?
![Screenshot 2023-11-20 at 3 12 52 PM](https://github.com/numaproj/numaflow/assets/4667739/d0d967c7-7a7d-45ab-be28-228b07db3b75)

Reply:
we are not using this `pipelineVertexMetricsFetch.ts`. so test is not needed. The API call made in this hook is also duplicated inside `fetcherHooks/pipelineViewFetch.ts`, so if we already have the tests lets add them to fetcherHooks/pipelineViewFetch.test.ts

Reply:
Gotcha!